apps/07tic-tac-toe/script.js

```diff
@@ -2,4 +2,170 @@
 
 $(document).on('ready', function() {
     // Put app logic in here
+    
+    var board = [
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
+];
+    
+    
+    function horizontalWin() {
+    if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn ||
+        board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn ||
+        board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+function verticalWin() {
+    if (board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn ||
+        board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn ||
+        board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+function diagonalWin() {
+    if (board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn ||
+        board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+    
+    function checkForWin() {
+    if (horizontalWin() || verticalWin() || diagonalWin()) {
+        return true
+        console.log('Player' + playerTurn + 'Won!');
```

Comment:
This console log will never happen as it is after a return statement. Remember, return statements will cause your function to stop running, so any code after the retrun statement in the function will not be run.